pallets/phala/src/registry.rs

```diff
@@ -337,6 +346,42 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// (called by a prouter on behalf of a worker)
+		#[pallet::weight(0)]
+		pub fn bind_worker_pnetwork_ident(
+			origin: OriginFor<T>,
+			pnetwork_ident_info: PhalaNetworkIdentBindingInfo,
```

Comment:
It's usually suggested to pass the raw args directly. No need to wrap in a struct.